kprogresshud/src/main/java/com/kaopiz/kprogresshud/KProgressHUD.java

```diff
@@ -372,6 +400,12 @@ protected void onCreate(Bundle savedInstanceState) {
             initViews();
         }
 
+        @Override
+        public void setOnKeyListener(OnKeyListener onKeyListener) {
+            super.setOnKeyListener(onKeyListener);
+            mOnKeyListener = onKeyListener;
+        }
```

Comment:
I think it not necessary to override this method, then save the reference which I don't see being call anywhere.